python/pyspark/streaming/kinesis.pyi

```diff
@@ -18,7 +18,7 @@
 
 # NOTE: This dynamically typed stub was automatically generated by stubgen.
```

Comment:
Nit. Could we drop this, as it is no longer automatically generated?

Reply:
dropped it, thanks :)